content/docs/csidriver/installation/test/certcsi.md

```diff
@@ -7,52 +7,137 @@ description: Tool to validate Dell CSI Drivers
 Cert-CSI is a tool to validate Dell CSI Drivers. It contains various test suites to validate the drivers. 
 
 ## Installation
-To install this tool you can download one of binary files located in [RELEASES](https://github.com/dell/cert-csi/releases)
 
-You can build the tool by cloning the repository and running this command:
+The recommended methods of installing `cert-csi` are downloading the exectuble from the latest GitHub Release or pulling the latest container image.
+
+> The exectuable from the GitHub Release only supports Linux. For non-Linux users, you must build the `cert-csi` executable [locally](#building-locally).
+
+### Download Release (Linux)
+
+1. Download the latest release of the cert-csi zip file.
+
 ```bash
-make build 
+curl -LO https://github.com/dell/cert-csi/releases/download/v1.3.0/cert-csi-v1.3.0.zip
```

Comment:
The examples for downloading the release are dependent on the 1.3.0 release, can this be generic so the docs don't require updates for every released version?

Reply:
I don't think it can be done that way unless the `cert-csi` asset name is also generic, but the release version is in it; e.g., `cert-csi-v1.2.0.zip`.